timemachine/fe/rbfe.py

```diff
@@ -537,10 +537,8 @@ def estimate_relative_free_energy_via_greedy_bisection(
 
     assert len(keep_idxs) <= n_windows
 
-    if min_overlap is not None and keep_idxs not in [None, [0, -1], list(range(n_windows))]:
-        raise ValueError(
-            "when min_overlap is not None, keep_idxs must be equal to one of None, [0, -1], or list(range(n_windows))"
-        )
+    if min_overlap is not None and not all(i in {0, -1} for i in keep_idxs):
```

Comment:
nit: Might be worth while to assert `len(keep_idxs) == 2` here as well since the current check allows `keep_idxs = [0, -1, -1]`

Reply:
Added an assertion of unique indices in https://github.com/proteneer/timemachine/pull/1115/commits/3388ef2cc181e973e049cacb0e0922d57d592a88